### ISSUE TITLE
Time not correct

### ISSUE THREAD

**Issue by cervedgroup** [osTicket/osTicket#831]
Server win2008r2 , time is 3.00 PM , GMT+1, Daylight saving time, location Italy

On osticket (1.8.1.2) - admin panel, date & time option , the system display 1.00 PM , same GMT same setting of Daylight saving time ... also changing the GMT osticket mantain incorrect time.

this bug display incorrect time when posting internal notes/reply/transfers ecc ...

**Comment by PeopleInside** [comment]
Time is wrong also for me and i continue to see this BUG. I had opened this: https://github.com/osTicket/osTicket-1.8/issues/1503 and still be valid also with the last udate 1.9.6

If i go on registration page (user side) the time is wrong. Now is 10.18 AM Paris and OsTicket Show 9.18 AM

Time is wrong also in Admin side in the settings.php

**When this will be fixxed?**
http://postimg.org/image/pdodtwh4j/
http://postimg.org/image/peybnbiyb/

**Comment by ntozier** [comment]
@PeopleInside did you read @greezybacon response?

**Comment by PeopleInside** [comment]
Umh yes read but you are right I had not gives a response. Sorry. Yes time is correct when I create a ticket. Is only not correct in the plaice I had tell you.

**Comment by ntozier** [comment]
@PeopleInside then you know that the times there are not intended to reflect the actual time, and are for intended for displaying the format of the times and dates that are entered in the boxes to the left of them as @greezybacon indicated.

**Comment by PeopleInside** [comment]
.. So why the time must be not the rigth time but one hours before the rigth time? Why is not the server time but is the wrong time? I could not understand. 

Is not a static example (luckily because no sense) is a wrong time one hours before the exact time.
The issue is not on Admin side but also in user side when user register to the support ticket. User will see wrong time so think server or system are not well configured. This is not good for the affidability and the work of the support help desk.

There are two position where this time is wrong, one in the admin side and the other one in user interface.

**Comment by ntozier** [comment]
You are correct that the time in the Account Registration screen is not the individual's correct time, my guess is its because the individual doesn't have an account yet to influence the offset.  So it looks like it's defaulting to GMT.

/include/client/register.inc.php
line 64
<strong><?php echo Format::date($cfg->getDateTimeFormat(),Misc::gmtime(),$info['tz_offset'],$info['dst']); ?></strong>)</em>

This appears to be a different issue than what this thread is about though.  This thread is specifically about the Admin panel date display section as noted in the first post, and several follow up threads.  You should probably start a new thread if you have a concern about the client side -> Account registration date/time display (if there isn't already one).

**Comment by PeopleInside** [comment]
Yes i will do new Thread about client side now. I think also the Admin side should be "fixxed" as for the visditor why in the Admin panel the time can't be correct as for the user?

No problem, Admin issue in this case is less than have wrong time in the user interface.
I see the title of this topic is Time not correct

Also if the user select other time different from Paris the time showed don't change (user side) so i think this part not work. I will open now new discussion as suggested.

https://github.com/osTicket/osTicket-1.8/issues/1842

**Comment by fictionedge** [comment]
I'm having similar problem of sorts. In my country we've entered the summer time (one less hour). I checked php.ini, the admin setting and checked the "Observe daylight savings " option. Every time anyone creates a ticked the time stamp is one hour late.

**Comment by cervedgroup** [comment]
same problem here with the DST change, 
only solution we found is to set the timezone to +2 for all agents (via ost_staff table)

**Comment by dmfweb** [comment]
I also have a similar issue, the interesting thing is that if I change my timezone to the wrong timezone it shows up correctly on the ticket dashboard, but then when I do an export of the tickets the times are wrong in the export...

**Comment by greezybacon** [comment]
@dmfweb you are right, currently, the times in the export are not localized. That is, they remain in the database timezone, whatever it is

**Comment by evandroamaro** [comment]
I'm using English version v1.9.12, and it also came with this error 'm' instead of 'i'. Now it's corrected. I didn't notice it myself. Thanks to this Topic and everyone who contributed.

**Comment by julianmatz** [comment]
I seem to be having the same issue. My server (and php.ini) is set to UTC. osTicket is set to GMT+0 (London, Dublin, Lisbon), with "Observe daylight savings" checked in System Settings as well as in My Preferences. It is currently still daylight savings (GMT+1), however, the time displayed in osTicket is GMT+0.

osTicket (v1.9.14)

**Comment by leandropc** [comment]
Using version 	v1.10 (901e5ea) and I have the same issue.

Seems likes it's related to the information shown in the **Database Information and Usage** section at yourdomain.com/scp/system.php

![screenshot_20170106_084451](https://cloud.githubusercontent.com/assets/17462646/21719551/6eb743ac-d3ec-11e6-84e5-223932c334e7.png)


It should be America/Havana

**Comment by Wohlraj** [comment]
Hello,
I have the same behavior as this last comment.
Database timezone is showed as UTC while everything else (php.ini, /etc/localtime) is set to Europe/Paris.

**Comment by balojs** [comment]
Add TimezoneField #3786

**Comment by andrelindner** [comment]
Can someone close the old tickets?

**Comment by JediKev** [comment]
@andrelindner We close issues when we are ready to close them.

**Comment by andrelindner** [comment]
> @andrelindner We close issues when we are ready to close them.

@JediKev Sure, you can close it when you're ready, but you should check the old tickets, not just forget them. Is it still an issue?
